pandas/_libs/groupby.pyx

```diff
@@ -778,47 +778,48 @@ def group_quantile(ndarray[float64_t] out,
             if not mask[i]:
                 non_na_counts[lab] += 1
 
-    # Get an index of values sorted by labels and then values
-    order = (values, labels)
-    sort_arr = np.lexsort(order).astype(np.int64, copy=False)
-
-    with nogil:
-        for i in range(ngroups):
-            # Figure out how many group elements there are
-            grp_sz = counts[i]
-            non_na_sz = non_na_counts[i]
+    if labels.any():
```

Comment:
What is this required for? Do we have a test case when this is False?

Reply:
Normally, if labels are false or the dataframe is empty, an error message will rise. However, there are case where certain operations leaves to empty labels, as when time resampling some types of empty dataframe:

Here is an example of the pipeline:
https://dev.azure.com/pandas-dev/pandas/_build/results?buildId=34258&view=logs&j=bef1c175-2c1b-51ae-044a-2437c76fc339&t=770e7bb1-09f5-5ebf-b63b-578d2906aac9&l=127

Reply:
Is this accounted for in the test that you've created? If not, can you add a test for it?

Reply:
add two cases that take the if labels.any(): path.

> What is this required for?

otherwise labels.max() can raise `ValueError: zero-size array to reduction operation maximum which has no identity`